pkg/config/agentmanagement.go

```diff
@@ -15,9 +15,13 @@ import (
 
 const cacheFilename = "remote-config-cache.yaml"
 
+var defaultConfig = AgentManagement{
+	CacheLocation: "data-agent/",
+}
+
 type labelMap map[string]string
 
-type RemoteConfiguration struct {
```

Comment:
While this works for now, I would still export it for consistency with other parts of the code where we define configuration types